.github/workflows/build-and-test.yml

```diff
@@ -12,13 +12,15 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v3
         with:
-          python-version: '3.6'
+          python-version: '3.8'
           cache: 'pip'
       - run: export PATH=$PATH:$HOME/.local/bin
       - run: export WAMP_BUILD_ID="$(date -u "+%Y%m%d")-$(git rev-parse --short ${GITHUB_SHA})"
+      - run: sudo apt-get update
       - run: sudo apt update
-      - run: sudo apt install -y xml2rfc libxml2-utils libenchant-2-2 enscript
+      - run: sudo apt install -y xml2rfc libxml2-utils libenchant-2-2 enscript weasyprint libpango-1.0-0 libharfbuzz0b libpangoft2-1.0-0 python3-cffi python3-brotli
       - run: pip install -r requirements.txt
+      - run: pip install 'weasyprint>=53.0'
```

Comment:
mmh, not sure, did I miss sth, why do we need weasyprint? the spec can be rendered into PDF using xml2rfc https://github.com/wamp-proto/wamp-proto/blob/master/Makefile#L92

Reply:
I faced that because python 3.6 is not supported within ubuntu-latest (2022) in GH CI. So I switched to 3.{7,8,9} and get all the fun %)